packages/react-dom-bindings/src/server/ReactDOMServerExternalRuntime.js

```diff
@@ -74,34 +69,47 @@ function installFizzInstrObserver(target /*: Node */) {
 }
 
 function handleNode(node_ /*: Node */) {
-  // $FlowFixMe[incompatible-cast]
-  if (node_.nodeType !== 1 || !(node_ /*: HTMLElement*/).dataset) {
+  if (
+    node_.nodeType !== 1 ||
+    // $FlowFixMe[incompatible-cast]
+    !(node_ /*: HTMLElement*/).dataset
+  ) {
     return;
   }
   // $FlowFixMe[incompatible-cast]
   const node = (node_ /*: HTMLElement*/);
   const dataset = node.dataset;
-  if (dataset['rxi'] != null) {
-    clientRenderBoundary(
-      dataset['bid'],
+
+  if (dataset['ix'] != null) {
```

Comment:
Question - would we expect / handle any of these instruction functions to throw here? I noticed a try-catch in `completeContainer`, but couldn't find what might throw in that function. Other instruction functions don't look like they would throw either

We should add a try-catch here if other functions can potentially throw (and React is expected to keep rendering). Inline scripts isolate failures, but the external runtime may skip processing further instructions / MutationRecords if one errors.

Reply:
I think I ended up removing the throw but not the wrapping try catch. Good call on handling it here though, we probably should not actually throw and just return from completeContainer